src/utils/getWADORSImageId.js

```diff
@@ -18,6 +18,7 @@ function getWADORSImageUrl(instance, frame) {
  * Obtain an imageId for Cornerstone based on the WADO-RS scheme
  *
  * @param {object} instanceMetada metadata object (InstanceMetadata)
+ * @param {(string\|number)} [frame] the frame number
```

Comment:
Do we need backslash here?

Reply:
I'm not super familiar with jsdocs but this was what I saw for allowing multiple types.